xbmc/guilib/DirtyRegion.h

```diff
@@ -35,3 +35,10 @@ class CDirtyRegion : public CRect
 };
 
 typedef std::vector<CDirtyRegion> CDirtyRegionList;
+
+class CRenderingStats
+{
```

Comment:
Any reason why this is part of the DirtyRegion header? Should we move it to its own header or move it to window manager header?